src/main/java/com/ua/accommodation/dto/accommodation/CreateAccommodationRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package com.ua.accommodation.dto.accommodation;
+
+import com.ua.accommodation.dto.accommodation.address.CreateAddressRequestDto;
+import com.ua.accommodation.dto.accommodation.amenity.CreateAmenityRequestDto;
+import com.ua.accommodation.model.Accommodation;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
+import java.math.BigDecimal;
+import java.util.Set;
+
+public record CreateAccommodationRequestDto(
+        @NotNull(message = "Type is required") Accommodation.Type type,
+
+        @Valid CreateAddressRequestDto location,
```

Comment:
if this works as it looks its good, but i cant see how we could send CreateAccommodationRequestDto with CreateAddressRequestDto as a parameter

Reply:
Please see in swagger on in my postman collection above

Reply:
Basically it look like:
`{
  "type": "HOUSE",
  "location": {
    "street": "Sunset Boulevard, 123",
    "city": "Los Angeles",
    "state": "California",
    "zipCode": "90210",
    "country": "United States"
  },
  "size": "Large",
  "amenities": [
    {
      "name": "Swimming Pool"
    },
    {
      "name": "Garden"
    },
    {
      "name": "Home Theater"
    }
  ],
  "dailyRate": 500,
  "availability": 10
}`